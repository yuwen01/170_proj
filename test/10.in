10 
99
0 1 1.163 67.233
0 2 83.073 4.797
0 3 14.067 91.341
0 4 32.144 81.093
0 5 19.560 33.705
0 6 5.707 17.374
0 7 59.038 14.420
0 8 10.811 18.280
0 9 14.691 10.111
1 2 15.612 22.652
1 3 26.892 90.090
1 4 27.661 35.095
1 5 48.711 47.782
1 6 73.781 41.570
1 7 16.484 83.385
1 8 81.505 58.229
1 9 92.745 13.952
2 3 55.212 56.368
2 4 3.296 19.560
2 5 23.946 62.753
2 6 61.434 47.504
2 7 73.798 35.408
2 8 9.761 34.596
2 9 4.391 27.080
3 4 84.344 24.311
3 5 44.339 4.661
3 6 9.502 93.513
3 7 76.523 33.550
3 8 81.171 16.290
3 9 54.225 75.395
4 5 12.460 72.503
4 6 51.221 54.630
4 7 20.399 87.497
4 8 85.523 21.893
4 9 85.325 71.270
5 6 19.814 33.683
5 7 43.132 2.858
5 8 77.196 55.940
5 9 68.720 82.022
6 7 2.341 44.595
6 8 15.858 57.779
6 9 96.241 99.793
7 8 36.240 10.495
7 9 56.308 49.118
8 9 89.374 99.158
