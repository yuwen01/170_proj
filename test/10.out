0 0
1 1
2 0
3 2
4 3
5 4
6 5
7 4
8 2
9 1
